### DISCUSSION TITLE
Support for Environments in Github Actions?

### DISCUSSION THREAD

**Discussion by stefandevo** [nuke-build/nuke#1388] [Q&A]
Wondering it would be possible to add support for Environments with Github Actions.

https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#using-an-environment

Basically, the `GitHubActions` attribute should support a kind of way to set the `environment` in the generated github action.

or is this already possible?

**Comment by stefandevo** [comment]
It would be easy for me to create a PR, but as I see the long list of open PR's I am not sure how quickly this would be merged... Is it still active maintained?

**Reply by ThomasBoettcherHolschen** [reply]
Yes, it is maintained. My Pull-Request was merged a few days alter it was finished.

**Comment by stefandevo** [comment]
https://github.com/nuke-build/nuke/pull/1389

**Reply by stefandevo** [reply]
Any news on an updated version with the outstanding PR's?
